# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1467914615 14400
#      Thu Jul 07 14:03:35 2016 -0400
# Node ID cf1bda5ca9c9d068b33fe78f5ce345d87586591b
# Parent  bd4f3d2ce52f87cf4ec4fe91c08391109cfba9a1
update list of gnulib modules

* bootstrap.conf (gnulib_modules):
Remove unused modules: pathmax, sleep.
Remove modules for functions now provided by standard C++:
copysign, copysignf, fclose, fflush, float, floor, floorf, fopen,
frexp, frexpf, log, log2, log2-ieee, log2f, log2f-ieee, logf,
mbrtowc, realloc-gnu, rename, round, roundf, stdint, trunc, truncf.

diff --git a/bootstrap.conf b/bootstrap.conf
--- a/bootstrap.conf
+++ b/bootstrap.conf
@@ -20,100 +20,74 @@
 # gnulib modules used by this package.
 gnulib_modules="
   areadlink
   base64
   canonicalize
   chdir
   close
   closedir
-  copysign
-  copysignf
   crypto/md2
   crypto/md4
   crypto/md5
   crypto/sha1
   crypto/sha256
   crypto/sha512
   dup2
-  fclose
   fcntl
-  fflush
   filemode
-  float
-  floor
-  floorf
   fnmatch
-  fopen
   fpucw
-  frexp
-  frexpf
   fseek
   ftell
   ftruncate
   getcwd
   gethostname
   getopt-gnu
   getrusage
   gettimeofday
   glob
   isatty
   largefile
   link
-  log
-  log2
-  log2-ieee
-  log2f
-  log2f-ieee
-  logf
   lstat
   malloc-gnu
-  mbrtowc
   mkdir
   mkfifo
   mkostemp
   mktime
   nanosleep
   nproc
   open
   opendir
-  pathmax
   pipe-posix
   progname
   putenv
   readdir
-  realloc-gnu
-  rename
   rmdir
-  round
-  roundf
   select
   sigaction
   signal
   sigprocmask
-  sleep
   stat
   std-gnu11
   stddef
-  stdint
   stdio
   strdup
   strerror
   strftime
   strptime
   strsignal
   symlink
   sys_stat
   sys_time
   sys_times
   sys_wait
   tempname
   tmpfile
-  trunc
-  truncf
   uname
   unistd
   unlink
   unsetenv
   vasprintf
   waitpid
 "
 
